www/js/share/components/reality-check/reality-check.controller.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * @name reality-check controller
+ * @author Nazanin Reihani Haghighi
+ * @contributors []
+ * @since 08/14/2016
+ * @copyright Binary Ltd
+ */
+
+(function() {
+    'use strict';
+
+    angular
+        .module('binary.share.components.reality-check.controllers')
+        .controller('RealityCheckController', RealityCheck);
+
+    RealityCheck.$inject = ['$scope', '$rootScope', '$state', '$timeout', '$location', 'websocketService', 'appStateService', 'accountService', '$ionicPopup', 'alertService', '$translate', 'languageService'];
+
+    function RealityCheck($scope, $rootScope, $state, $timeout, $location, websocketService, appStateService, accountService, $ionicPopup, alertService, $translate, languageService) {
+			var vm = this;
+			var landingCompanyName;
+			$scope.$on('authorize', function(e, authorize) {
```

Comment:
Would you please follow ES6 code syntax and use arrow function in case of needing callback function.